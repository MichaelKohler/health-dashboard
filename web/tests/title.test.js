/* global it, expect */
/* eslint-disable react/jsx-filename-extension */

import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Title from '../src/title.jsx';

it('renders correctly', () => {
  const wrapper = shallow(<Title>Hi!</Title>);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});

/* eslint-enable react/jsx-filename-extension */
