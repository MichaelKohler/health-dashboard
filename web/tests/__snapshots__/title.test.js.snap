// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<WithStyles(ForwardRef(Typography))
  color="primary"
  component="h2"
  gutterBottom={true}
  variant="h6"
>
  Hi!
</WithStyles(ForwardRef(Typography))>
`;
